tests/foreman/api/test_reporttemplates.py

```diff
@@ -811,8 +715,11 @@ def test_positive_generate_job_report(setup_content, module_target_sat, content_
             'input_values': {"job_id": job["id"]},
         }
     )
-    assert res[0]['Host'] == content_hosts[0].hostname
-    assert res[1]['Host'] == content_hosts[1].hostname
+    assert (
+        res[0]['Host'] == content_hosts[0].hostname and res[1]['Host'] == content_hosts[1].hostname
+    ) or (
+        res[0]['Host'] == content_hosts[1].hostname and res[1]['Host'] == content_hosts[0].hostname
+    )
```

Comment:
Let's use sets to check if all hosts were reported. 
```suggestion
    assert {i.Host for i in res} == {i.hostname for i in content_hosts}
```